src/test/java/com/webank/webase/front/util/TransactionEncoderTest.java

```diff
@@ -0,0 +1,96 @@
+package com.webank.webase.front.util;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.math.BigInteger;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+import org.fisco.bcos.web3j.crypto.*;
+import org.fisco.bcos.web3j.crypto.gm.GenCredential;
+import org.fisco.bcos.web3j.rlp.RlpType;
+import org.fisco.bcos.web3j.utils.Numeric;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class TransactionEncoderTest {
+
+   public static BigInteger gasPrice = BigInteger.ONE;
+   public static BigInteger gasLimit = BigInteger.TEN;
+
+    @Test
+    public void testSignMessage() {
+        byte[] signedMessage =
+                TransactionEncoder.signMessage(createContractTransaction(), SampleKeys.CREDENTIALS);
+        String hexMessage = Numeric.toHexString(signedMessage);
+        assertThat(
+                hexMessage,
+                is(
+                        "0xf85a8201f4010a8201f5840add5355887fffffffffffffff801ba01cf44d4680e1ecaf11a9a997b08055ae84c5d417b1fc7c2bdbaffc3fd4a7659aa07a424ef2ad019c599a24309c97f4cd10d0e4293a51d8c1abb095052bf54a7ba7"));
+    }
+
+
+    @Test
+    public void testGMSignMessage() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)